src/project-tests/survey-form-tests.js

```diff
@@ -116,6 +116,13 @@ export default function createSurveyFormTests() {
 
             it('12. I can select an option from a dropdown that has corresponding id="dropdown".', function() {
                 var dropdown = document.getElementById('dropdown');
+				var dropdownid = dropdown.getAttribute('id');
```

Comment:
I don't think we need this line? In the line above we are already getting the same element and specifying the id. But I do think it's a good idea to not repeat the string 'dropdown' several times. I would refactor as follows:

```
var dropdownid = 'dropdown';
var dropdown = document.getElementById(dropdownid);
etc.
```